package.json

```diff
@@ -48,6 +48,7 @@
     "@types/express-serve-static-core": "4.17.28",
     "@types/fast-json-stable-stringify": "2.0.0",
     "@types/jest": "27.4.1",
+    "@types/keyv": "^3.1.3",
```

Comment:
I'm not sure why it would be. It _is_ added as a runtime dependency in server, is that what you mean?

Reply:
Hmm. So the runtime dependencies are given as caret ranges typically; I thought that we generally also pinned a specific version at the top level, so that Renovate can keep us up to date by updating package.json rather than just package-lock.json (assuming we don't want Renovate to actually bump the minimum on the real package's caret). Maybe I'm confused though.

Reply:
Had to stew on this a bit to understand what you were after. This actually makes a lot of sense and I don't think we've ever done it this way before. I imagine there are a number of packages where we would want to also add to the top-level. Maybe we want to do this comprehensively with all of server's (and server-type's) dependencies?